users/templates/users/password_reset_email.html

```diff
@@ -0,0 +1,18 @@
+<html>
+<body>
+<p>Hi <strong>{{ user.username }}</strong>,</p>
+
+<p>Please click on the link below to reset your password.</p>
+
+<p><a href="http://localhost:8000{% url 'users:password_reset' token=user.password_reset_token %}">Reset password</a></p>
+
+<p>or copy this link</p>
+
+<p>http://localhost:8000{% url 'users:password_reset' token=user.password_reset_token %}</p>
```

Comment:
Done! In der confirmation email war das auch noch drin, habe beides rausgenommen.